v3/internal/commands/build_assets/Taskfile.linux.yml

```
@@ -48,6 +52,48 @@ tasks:
+  create:aur:
+    summary: Creates a arch linux packager package
+    deps:
+      - task: build
+        vars:
+          PRODUCTION: "true"
+    cmds:
+      - task: generate:aur
+
+  generate:deb:
+    summary: Creates a deb package
+    cmds: 
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)